testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/federation/kerberos/AbstractKerberosSingleRealmTest.java

```diff
@@ -216,4 +213,53 @@ public void credentialDelegationTest() throws Exception {
 
         events.clear();
     }
+
+    @Test
+    public void credentialDelegationUserInfoTest() throws Exception {
```

Comment:
Is it possible to use existing method `credentialDelegationTest()` and just add the call to userInfo to that method?

I know that you need to use AccessTokenResponse, but you can possibly just use your new method to retrieve `AccessTokenResponse` and then do simply:
```
AccessToken token = oauth.verifyToken(tokenResponse.getAccessToken());
```
And then you have both `tokenResponse` and `token` though?

Reply:
Done, integrated the test into the existing `credentialDelegationTest()`